examples/react-router-4/components/App.js

```diff
@@ -28,9 +35,10 @@ function App({ logout }) {
           {' '}
           <Link to="/admin">{'Admin'}</Link>
           {' '}
-          <Link to="/login">Login</Link>
-          {' '}
-          <button onClick={() => logout()}>Logout</button>
+          {user.data ?
+            <button onClick={() => logout()}>Logout</button> :
+            <Link to="/login">Login</Link>
+          }
```

Comment:
To be honest, I only found this project last night and have not had much of a chance to look at all the API, I agree examples are a good chance to show that stuff, I'll take a look and see what I can manage here.